robot_skills/src/robot_skills/arms.py

```diff
@@ -618,7 +618,7 @@ def send_joint_trajectory(self, configuration, timeout=5.0, max_joint_vel=0.7):
 
     def reset(self, timeout=0.0):
         """
-        Put the arm into the 'reset' pose
+        Put the arm into the 'reset' pose. And open the gripper
```

Comment:
How about setting the occupied_by to None?

Reply:
Opening the gripper generally results in that.

Reply:
Real life discussion with Matthijs: occupied by = None should be included in open gripper then. 
Technically occupied by should not be in the robot object.